client/src/lib/templates/LandingTemplate/LandingTemplate.svelte

```diff
@@ -24,14 +27,23 @@
         <div class="fr-pl-3w">
           <h1>Bienvenue sur le service de catalogage de données de l’État</h1>
           <p>
-            Ce service permet de favoriser la circulation des données entre les
-            différentes administrations centrales et opérateurs de l’État.
-            L’intégration des différentes organisations se fait progressivement
-            et en fonction des demandes.
+            Ce service permet aux administrations centrales et aux opérateurs
+            sous leur tutelle de créer, gérer et ouvrir leurs catalogues de
+            données dans le cadre notamment de leur stratégie en matière de
+            politique de la donnée. Votre organisation doit être enregistrée
+            pour vous identifier.
           </p>
 
           <div class="button-container">
             <MonCompteProButton on:click={triggerDataPassLoginFlow} />
+            <div>
+              <a
+                target="_blank"
```

Comment:
Avec `target="_blank"` il faut utiliser `rel="noopener"` pour éviter le tabsnapping

https://techstacker.com/html-noopener-attribute/

```suggestion
                target="_blank"
                rel="noopener"
```